src/feditest/nodedrivers/mastodon/api.py

```diff
@@ -0,0 +1,295 @@
+            if key in value:
+                value = value.get(key)
+            elif default or not required:
+                return default
+            else:
+                raise KeyError(f"Can't get node parameter for {keys}")
+        return value
+
+    # Override
+    def obtain_actor_document_uri(self, actor_rolename: str | None = None) -> str:
+        return (
+            self.actors_by_role[actor_rolename]
+            if actor_rolename
+            else self.default_actor
+        ).uri
+
+    # Override
+    def obtain_followers_collection_uri(self, actor_uri: str | None = None) -> str:
+        """
+        Determine the URI that points to the provided actor's followers collection.
+        """
+        return self._get_actor_or_default(actor_uri).actor["followers"]
+
+    # Override
+    def obtain_following_collection_uri(self, actor_uri: str | None = None) -> str:
+        """
+        Determine the URI that points to the provided actor's following collection.
+        """
+        return self._get_actor_or_default(actor_uri).actor["following"]
+
+    # Override
+    # FIXME: The base API doesn't have a `to` kwargs.
+    # However, delivery will not happen without recipients.
+    # We could possibly default this to the followers URI.
+    # One of the tests also uses another kwarg to specify the preferred
+    # inbox type.
+    def make_create_note(
+        self, actor_uri: str, content: str, to: str | None = None
+    ) -> str:
+        """ "
+        Perform whatever actions are necessary to the actor with actor_uri will
+        have created a Note on this Node. Determine the various Note and
+        Activity properties as per this Node's defaults.
+        """
+        sending_account = self._get_actor_or_default(actor_uri)
+        assert sending_account.is_mastodon
+        if to:
+            results = sending_account.api.search(q=to, result_type="accounts")
+            if to_account := next(
+                (a for a in results.get("accounts", []) if a.uri == to), None
+            ):
+                to_url = urlparse(to_account.uri)
+                to_handle = f"@{to_account.acct}@{to_url.netloc}"
+                content += f" {to_handle}"
+        response = sending_account.api.status_post(content)
+        self.uri_map[response.uri] = response
+        return response.uri
+
+    # Override
+    def wait_for_object_in_inbox(self, actor_uri: str, note_uri: str) -> str:
+        for _ in range(
+            self._param("inbox_wait", "retry_count", default=5)
+        ):  # TODO Try for 5 seconds. Make configurable?
+            response = next(
+                (
+                    s
+                    for s in self._get_actor_or_default(actor_uri).api.timeline("local")
+                    if s.uri == note_uri
+                ),
+                None,
+            )
+            if response:
+                return
+            time.sleep(self._param("inbox_wait", "retry_count", default=1))
```

Comment:
I think this wants to be `"inbox_wait", "retry_interval"`.

Reply:
> I think this wants to be `"inbox_wait", "retry_interval"`.

Done.